CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1656,6 +1767,17 @@ local function OnVREnabled(prop)
 			end
 		end)
 
+		-- Turn off dragging when in VR
+		for _, slot in pairs(Slots) do
+			slot:SetClickability(false)
+		end
+
+		-- Reset the constants!
```

Comment:
Right, I will address this by doing some updates that unlink the backpack from the VR interface